content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -49,6 +49,13 @@ RuntimeClass in the **node.k8s.io/v1beta1** API version will no longer be served
 * All existing persisted objects are accessible via the new API
 * No notable changes
 
+#### PodSecurityPolicy {#psp-v125}
+
+PodSecurityPolicy in the **policy/v1beta1** API version will no longer be served in v1.25, and the PodSecurityPolicy admission controller will be removed.
```

Comment:
```suggestion
The **policy/v1beta1** API version of PodSecurityPolicy will no longer be served in v1.25, and the PodSecurityPolicy admission plugin will be removed.
```

Reply:
Would be good to add this, but I don't see it blocking a merge.